src/main/telemetry/jetiexbus.c

```diff
@@ -299,32 +387,37 @@ void handleJetiExBusTelemetry(void)
     }
 }
 
-void sendJetiExBusTelemetry(uint8_t packetID)
+
+
+uint8_t sendJetiExBusTelemetry(uint8_t packetID, uint8_t item)
 {
-    static uint8_t sensorDescriptionCounter = 0;
-    static uint8_t sensorValueCounter = 1;
+    static uint8_t sensorDescriptionCounter = 0xFF;
     static uint8_t requestLoop = 0;
     uint8_t *jetiExTelemetryFrame = &jetiExBusTelemetryFrame[EXBUS_HEADER_DATA];
-
-    if (requestLoop == 100) {              //every nth request send the name of a value
-        if (sensorDescriptionCounter == JETI_EX_SENSOR_COUNT )
-            sensorDescriptionCounter = 0;
-
+    
+    if (requestLoop < 0xFF) {
+        while(1) {
```

Comment:
This would be clearer if the break conditions where used with the `while`.

Reply:
It's done.